sonic_installer/main.py

```diff
@@ -576,6 +577,16 @@ def install(url, force, skip_platform_check=False, skip_migration=False, skip_pa
                 "Aborting...", LOG_ERR)
             raise click.Abort()
 
+        # Calling verification script by default - signature will be checked if enabled in bios
+        echo_and_log("Verifing image {} signature...".format(binary_image_version))
+        try:
+            if not bootloader.verify_image_sign(image_path):
```

Comment:
If the force option is enabled, can we skip the verify_image_sign, as descirption in line 514? The check logic should be the same behavior as line 567 (secure bot), right?

Reply:
@xumia we don't want to allow force option for this feature, it will reduce its security properties. This remark was addressed in the previous merged and reverted PR https://github.com/sonic-net/sonic-utilities/pull/2337 (at least as I can remember)

Reply:
I agree with @xumia.
The security is protected by the secure boot process, not by the installer. If the user intention is to install it anyway, no matter it will boot or not, we should respect the intetion, right?

Reply:
But this is not what was defined in the HLD for this feature.
If we would like to make a feature change for this item - it should go as a separate request.